packaging_automation/tests/test_package_cloud_statistics_collector.py

```diff
@@ -20,15 +23,24 @@
 def test_fetch_and_save_package_cloud_stats():
     db = create_engine(db_connection_string(db_params=db_parameters, is_test=True))
     db.execute(text(f'DROP TABLE IF EXISTS {PackageCloudDownloadStats.__tablename__}'))
-    pack_count = package_count(ORGANIZATION, REPO, PACKAGE_CLOUD_API_TOKEN)
     session = db_session(db_params=db_parameters, is_test=True)
     page_record_count = 3
     parallel_count = 3
+
+    result = stat_get_request(
+        package_list_with_pagination_request_address(PACKAGE_CLOUD_API_TOKEN, 1, ORGANIZATION, REPO, 100),
+        RequestType.package_cloud_list_package, session)
+    package_info_list = json.loads(result.content)
+    package_list = list(filter(
+        lambda p: not p["name"].endswith(("debuginfo", "dbgsym")) and not p["name"].startswith(
+            ("citus-ha-", "pg-auto-failover-cli")),
```

Comment:
Can you use the `is_ignored_package` function here?

Reply:
Yes I think we can use fixed as you suggested

Reply:
Fixed